### DISCUSSION TITLE
How run Mesa example in Replit

### DISCUSSION THREAD

**Discussion by josse7cz** [mesa/mesa#1284] [Q&A] [answered]
Hello, is it possible to give me advice on how to run mesa examples on Replit.com? Thank you
https://replit.com/@JosefJanda/mesa-1#examples/

**Comment by josse7cz** [comment, marked as answer]
Muss add to .replit file
language = "python3"
run = "python src/main.py" <=HERE IS inner directory path

**Reply by rht** [reply]
I tried https://replit.com/@JosefJanda/mesa-1#examples/ and it worked. Does that mean you have solved the problem?

**Reply by rht** [reply]
This is the actual .replit content
```
language = "python3"
run = "python examples/schelling/run.py"
```

**Reply by rht** [reply]
@josse7cz are you open to documenting what you did in a file called running_on_replit.rst inside this folder: https://github.com/projectmesa/mesa/tree/main/docs/useful-snippets ? This way, more people will know about it. It seems to be faster than Google Colab because the Mesa has been pre-installed once for all. Though the use case is a bit different, since only 1 person can edit the content of the files in the replit. @tpike3 you might want to check this out.

**Reply by rht** [reply]
Side note: I have looked around if it is possible to do something like `pip install` and `npm run build` at the same time within one env in either Colab/Replit. It looks like only Colab supports it (because you can do `!npm ...` inside the Jupyter).

**Comment by josse7cz** [comment]
Yes.  It is so.S pozdravem Josef Janda.
